content/en/logs/guide/azure-automated-log-forwarding.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Azure Automated Log Forwarding Setup
+private: true
+further_reading:
+- link: "https://www.datadoghq.com/blog/monitoring-azure-platform-logs/"
+  tag: "Blog"
+  text: "Best practices for monitoring Microsoft Azure platform logs"
+---
+
+## Overview
+
+<div class="alert alert-info">Azure automated log forwarding is in Preview.</div>
+
+Use this guide to automate your Azure log forwarding setup with an Azure Resource Manager (ARM) template. 
+
+The ARM template deploys resources from a series of Azure services (storage accounts and function apps) into your subscriptions that collect and forward logs to Datadog. These services automatically scale up or down to match log volume. Scaling is manged by a control plane, which is a set of function apps deployed to a subscription and region of your choice. Storage accounts and function apps are deployed in each of the subscriptions forwarding logs to Datadog.
+
+**All sites**: All [Datadog sites][4] can use the steps on this page to complete setup of automated log forwarding.
```

Comment:
I had some trouble understanding what this meant with "Datadog sites" being the subject of the verb "use" here. Maybe something like:

```suggestion
**All sites**: Automated log forwarding is available to use on all [Datadog sites][4].
```